src/frontend/src/components/AccessibilityStatement/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { HashLink as Link } from "react-router-hash-link";
+
+class AccessibilityStatement extends React.Component {
+
+  componentDidMount(){
+    document.title = "Accessibility Statement - RecordSponge"
```

Comment:
Why does this need to be updated? Also should we add this or `<title>` tags on our other views?

Reply:
[This article](https://www.24a11y.com/2018/the-d-in-the-dom/) explains a few reasons why, and I'm going to update the rest in a separate issue.